examples/case_studies/BART_quantile_regression.myst.md

```diff
@@ -0,0 +1,171 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+(BART_quantile)=
+# Quantile Regression with BART
+:::{post} Jan 25, 2023
+:tags: BART, non-parametric, quantile, regression 
+:category: intermediate, explanation
+:author: Osvaldo Martin
+:::
+
+```{code-cell} ipython3
+from pathlib import Path
+
+import arviz as az
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import pymc as pm
+import pymc_bart as pmb
+
+from scipy import stats
+
+print(f"Running on PyMC v{pm.__version__}")
+```
+
+```{code-cell} ipython3
+%config InlineBackend.figure_format = "retina"
+RANDOM_SEED = 5781
+np.random.seed(RANDOM_SEED)
```

Comment:
Is this for the bart submodule/library not using generators/random states yet?

Reply:
Yes